README.md

```diff
@@ -81,6 +81,36 @@ Will search Spire for a customer that matches the given query
 Spire::Customer.search('casey.li@bitesite.ca')
 ```
 
+### Orders
+The syntax for orders is very similar to items. For additional information please refer to `lib/order.rb`.
+
+Will retrieve one order from Spire
+```ruby
+Spire::Order.find(orderId)
+```
+
+Will search Spire for any order that match the given query
+```ruby
+Spire::Order.search('casey.li@bitesite.ca')
+```
+
+Will create order in Spire   
+```ruby
+Spire::Order.create({
+  'customer' => {'id': 'customer id'},
+  'address' => { 
+    'streetAddress':'555\n BILLING ADDR street\n NO APT'
+  },
+  'shippingAddress' => { 
+    'streetAddress':'555\n SHIPPING ADDR street\n NO APT'
+  },
+  'items' => {
+    'inventory': {'id': 'inventory id'}
+  }
```

Comment:
let's have line 1-4 added here instead of streetAddress since it is being deprecated.

Reply:
let's also add all the other params in order to create an order, as we know that an order is not really valid without freight. we can do something like this:

```

{
  "customer": { "id": 3031 },
  "address": {
    "streetAddress": "555\n BILLING ADDR street\n NO APT"
  },
  "shippingAddress": {
    "streetAddress": "123\n SHIPPING ADDR street\n APT111"
  },
  "items": [
    { "inventory": { "id": 93 }, "orderQty": "2" },
    { "inventory": { "id": 2279 } },
    {
      "description": "MAKE COMMENT THRU API",
      "comment": "MAKE COMMENT THRU API"
    }
  ],
  "discount": "4",
  "freight": "14"
}
```

Reply:
Sounds good!! thanks for putting it together!